lib/game.rb

```diff
@@ -65,32 +62,36 @@ def get_coordinates
   end
 
   def get_row
-    @display.valid_row(Math.sqrt(dimension_of_board))
+    @display.valid_row(@board.size_of_board)
   end
 
   def get_column
-    @display.valid_column(Math.sqrt(dimension_of_board))
+    @display.valid_column(@board.size_of_board)
   end
 
-  def state_of_board(board, mark)
-    @board.mark_board(board, @row, @column.to_i, mark)
+  def state_of_board(mark)
```

Comment:
what is the behaviour of this method?  Is it to return the state of the board or does it do anything else?  I'd name it wrt what it is doing.